classes/class-wc-connect-paypal-ec.php

```diff
@@ -71,14 +71,19 @@ public function init() {
 
 		public function register_refund_pointer( $pointers ) {
 			$target = '.refund-actions > button:first-child';
-			$link_url = wc_gateway_ppec()->ips->get_signup_url( wc_gateway_ppec()->settings->environment );
 			$pointers[] = array(
 				'id' => 'wc_services_refund_via_ppec',
 				'target' => $target,
 				'options' => array(
 					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
 						__( 'Link a PayPal account' ,'woocommerce-services' ),
-						sprintf( __( "To issue refunds via PayPal Express Checkout, you will need to <a href=\"%s\">link a PayPal account</a> with the email address that received this payment.", 'woocommerce-services' ), $link_url )
+						sprintf(
+							wp_kses(
+								__( 'To issue refunds via PayPal Express Checkout, you will need to <a href=\"%s\">link a PayPal account</a> with the email address that received this payment.', 'woocommerce-services' ),
```

Comment:
You don't need to escape the double quotes now that the string is single quoted.